src/common/streamwriter.h

```diff
@@ -34,20 +35,30 @@ class StreamWriter {
     void putInt64(int64_t val);
     void putCString(const std::string &str);
 
+    void write(char obj); // alias for putByte
```

Comment:
Same: `put`, `write` and `<<` do essentially the same thing. I don't see why is that necessary.